core/src/main/java/com/linecorp/armeria/server/ServiceRequestContext.java

```diff
@@ -242,8 +235,7 @@ default SafeCloseable push(boolean runCallbacks) {
      * <pre>{@code
      * ctx.onChild((curCtx, newCtx) -> {
      *     assert ctx == curCtx && curCtx != newCtx;
-     *     // Add a callback to the child context.
-     *     newCtx.onExit(() -> { ... });
+     *     ...
```

Comment:
It's kinna difficult for me to show a proper example for this right now. Do you have any thing recommended?
Or can we remove this method as well? 🤔

Reply:
I'm not sure if we can remove this method or not. It seems like it's not used anywhere in our code, though. Slightly in favor of removal.

Reply:
We used it before adding attr lookup from root context only for copying attributes. I can't think of a good use case not better using the root lookup so in favor of removing

Reply:
Let me remove this as well. 😄